spec.bs

```diff
@@ -1699,6 +1704,45 @@ Several APIs specific to fenced frames are defined on the {{Fence}} interface.
   </wpt>
 </div>
 
+<div algorithm>
+  The <dfn method for=Fence>notifyEvent(|event|)</dfn> method steps are:
+
+  1. [=Assert=]: [=this=]'s {{Document}} is [=fully active=].
+
+  1. Let |navigable| be [=this=]'s [=relevant global object=]'s [=navigable=].
+  
+  1. If any of the following conditions are met, throw a {{SecurityError}} {{DOMException}}:
+
+      1. |navigable| is not a [=fenced navigable container/fenced navigable=].
```

Comment:
This should be an unordered list of conditions with semi-colons after each line except for the last. See https://html.spec.whatwg.org/multipage/browsing-the-web.html#attempt-to-populate-the-history-entry's-document:~:text=Otherwise%2C%20if%20all%20of%20the%20following%20are%20true for example.

Reply:
Done.